### PR TITLE
Feature/start service

### PR DESCRIPTION
**Pull request by wesleymonte**

### CODE REVIEW

File: bin/stop-service.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+DIRNAME=`dirname $0`
+cd $DIRNAME/..
+
+readonly ARREBOL_PORT=8080
+
```
Comment (thiagomanel) [inline bin/stop-service.sh]: is it the usual way to stop things in the context of springboot apps? I think lsof'ing things is not very reliable (at least, not much readable). Maybe, using pgrep or writing the pid of the process after calling the start script

**Review by thiagomanel — CHANGES_REQUESTED** [review]
changes are not mandatory, I'm satisfied with a reasonable answer to the point raised

File: bin/stop-service.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+DIRNAME=`dirname $0`
+cd $DIRNAME/..
+
+if [ ! -f ./bin/shutdown.pid ]; then
+    echo "shutdown.pid not found!"
+else
```
Comment (thiagomanel) [inline bin/stop-service.sh:8]: nice @wesleymonte. What if we do start-stop twice? the shutdown file keeps more than one pid?

**Review by thiagomanel — CHANGES_REQUESTED** [review]
almost done

File: bin/stop-service.sh (hunk 2 above)
Comment (wesleymonte) [reply bin/stop-service.sh:8]: No @thiagomanel, when the service is stopped the shutdown.pid is automatically removed.

Comment (thiagomanel) [reply bin/stop-service.sh:8]: cool, @wesleymonte. thanks

**Review by thiagomanel — APPROVED** [review]

**Pull request merged**